README.md

```diff
@@ -1,31 +1,46 @@
 
 ## Using this project
-As outlined briefly above, both this repo and the `apollo-frontend` projects need to be provided with the `APOS_EXTERNAL_FRONT_KEY` environment variable set to the same value. Then, you can start the project as you would any ApostropheCMS project. For example, in a local development environment you can use `npm run dev`. For more details on working with the code in an ApostropheCMS project you can examine our [documentation](https://docs.apostrophecms.org/).
 
-### Similarities to a stand-alone project
-If you have worked with an ApostropheCMS project previously, this repo should look as expected. There are a number of custom modules, providing new pieces, pages, and widgets, located in the `modules` folder. There are also a number of modules that are improving the core Apostrophe modules located in the `modules/@apostrophecms` folder. For a full understanding of Apostrophe you should consult the [documentation](https://docs.apostrophecms.org/), but we will touch on a few highlights later in this document.
+### Prerequisites
+- Node.js v20 or later
+- MongoDB v4.4 or later running on a local server or access to Atlas. See the [ApostropheCMS documentation](https://docs.apostrophecms.org/guide/development-setup.html) for set-up details.
+
+The codebases located in the `backend` and `frontend` folders should be treated as interlinked but separate projects.
+- To start, open a terminal instance at the root of each folder and run `npm install`.
+- Next, each project needs to be provided with an `APOS_EXTERNAL_FRONT_KEY` environment variable set to the same string value in order to authenticate communication. For example, in each terminal execute `export APOS_EXTERNAL_FRONT_KEY=my-secret-key`.
+- The `astro.config.mjs` file is already set to the normal default values, but if you are running the backend server on a different port, you will also have to set the `APOS_HOST` environment variable.
+- Then, you can start the projects using the accompanying scripts. For example, in a local development environment you can start each with `npm run dev`.
+
+### Similarities to a stand-alone ApostropheCMS project
+If you have worked with an ApostropheCMS project previously, the backend repo should look as expected. There are a number of custom modules, providing new pages, pieces, and widgets, located in the `modules` folder. There are also a number of modules that are improving the core Apostrophe modules located in the `modules/@apostrophecms` folder. For a full understanding of Apostrophe you should consult the [documentation](https://docs.apostrophecms.org/), but we will touch on a few highlights later in this document.
 
 Like any ApostropheCMS project, after creating a new module it needs to be registered in the `app.js` file. For pages or 'piece-type-pages' they also need to be registered in the `modules/@apostrophecms/page/index.js` file.
```

Comment:
How about just "Page types need to be added to the `types` option of the..."

Reply:
Changed.